gamemode/core/libraries/prone.lua

```
@@ -1,2 +1,7 @@
+--[[
+    Compatibility helpers for the Prone Mod.
+    Automatically exits the prone state when a player dies or
+    loads a new character so players don't get stuck.
+]]
 ﻿hook.Add("DoPlayerDeath", "PRONE_DoPlayerDeath", function(client) if client:IsProne() then prone.Exit(client) end end)
```

Luacheck (1.2.0)

[error] expected statement near '\u{FEFF}'

(E011)